VEDA_OS01/transcripts.py

```diff
@@ -0,0 +1,207 @@
+"""
+Transcript handlers.
+"""
+import logging
+from uuid import uuid4
+
+import boto
+import django.dispatch
+import requests
+from boto.s3.key import Key
+from pysrt import SubRipFile
+from rest_framework import status
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from control.veda_val import VALAPICall
+from VEDA_OS01 import utils
+from VEDA_OS01.models import (TranscriptPreferences, TranscriptProcessMetadata,
+                              TranscriptProvider, TranscriptStatus,
+                              VideoTranscriptStatus)
+
+requests.packages.urllib3.disable_warnings()
+
+LOGGER = logging.getLogger(__name__)
+
+VALID_TRANSCRIPT_PROVIDERS = [
+    '',
+    '',
+]
+TRANSCRIPT_SJSON = 'sjson'
+CIEOLO24_TRANSCRIPT_COMPLETED = django.dispatch.Signal(providing_args=['job_id', 'lang_code', 'org', 'video_id'])
```

Comment:
do we need signal? can we afford this extra? as we can easily follow modular approach without it.